### PR TITLE
feat: maven archetype for Spring SDK with Kotlin

### PR DESCRIPTION
**Pull request by octonato**
Only the archetype for now. No docs, no samples. We can experiment a bit with it and eventually produce more content.

### CODE REVIEW

File: maven-java/kalix-spring-boot-kotlin-archetype/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+For the latest release see [GitHub releases](https://github.com/lightbend/kalix-jvm-sdk/releases).
+
+To kickstart a project on Linux and macOS:
+
+```shell
+mvn \
+  archetype:generate \
+  -DarchetypeGroupId=io.kalix \
+  -DarchetypeArtifactId=kalix-spring-boot-kotlin--archetype \
```
Comment (efgpinto) [inline maven-java/kalix-spring-boot-kotlin-archetype/README.md:13]:
```suggestion
  -DarchetypeArtifactId=kalix-spring-boot-kotlin-archetype \
```

File: maven-java/kalix-spring-boot-kotlin-archetype/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  -DarchetypeVersion=LATEST
+```
+
+To kickstart a project on Windows 10 or later:
+
+```shell
+mvn ^
+  archetype:generate ^
+  -DarchetypeGroupId=io.kalix ^
+  -DarchetypeArtifactId=kalix-spring-boot-kotlin--archetype ^
```
Comment (efgpinto) [inline maven-java/kalix-spring-boot-kotlin-archetype/README.md:23]:
```suggestion
  -DarchetypeArtifactId=kalix-spring-boot-kotlin-archetype ^
```

File: maven-java/kalix-spring-boot-kotlin-archetype/src/main/resources/archetype-resources/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+
+## Designing
+
+To understand the Kalix concepts that are the basis for this example, see [Designing services](https://docs.kalix.io/services/development-process.html) in the documentation.
+
+
+## Developing
+
+This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services](https://docs.kalix.io/services/). Spring-SDK is an experimental feature and so far there is no [official](https://docs.kalix.io/) documentation. Examples can be found [here](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples) in the folders with "spring" in their name.
```
Comment (efgpinto) [inline maven-java/kalix-spring-boot-kotlin-archetype/src/main/resources/archetype-resources/README.md:11]: Not so experimental anymore and we actually have docs \o/

**Review by efgpinto — APPROVED** [review]
LGTM, left some minor comments.

File: maven-java/kalix-spring-boot-kotlin-archetype/src/main/resources/archetype-resources/README.md (hunk 3 above)
Comment (octonato) [reply maven-java/kalix-spring-boot-kotlin-archetype/src/main/resources/archetype-resources/README.md:11]: Hmm, we should adapt the spring boot java archetype as well.

File: maven-java/kalix-spring-boot-archetype/src/main/resources/archetype-resources/README.md [hunk 4]
```diff
@@ -1,15 +1,12 @@
...
 ## Designing
 
 To understand the Kalix concepts that are the basis for this example, see [Designing services](https://docs.kalix.io/services/development-process.html) in the documentation.
 
-
 ## Developing
 
-This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services](https://docs.kalix.io/services/). Spring-SDK is an experimental feature and so far there is no [official](https://docs.kalix.io/) documentation. Examples can be found [here](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples) in the folders with "spring" in their name.
-
+This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services](https://docs.kalix.io/services/) and check Spring-SDK [official documention](https://docs.kalix.io/spring/index.html). Examples can be found [here](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples) in the folders with "spring" in their name.
```
Comment (efgpinto) [inline maven-java/kalix-spring-boot-archetype/src/main/resources/archetype-resources/README.md:9]: My bad 😬 

```suggestion
This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services](https://docs.kalix.io/services/) and check Spring-SDK [official documentation](https://docs.kalix.io/spring/index.html). Examples can be found [here](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples) in the folders with "spring" in their name.
```

File: maven-java/kalix-spring-boot-kotlin-archetype/src/main/resources/archetype-resources/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# ${artifactId}
+
+## Designing
+
+To understand the Kalix concepts that are the basis for this example, see [Designing services](https://docs.kalix.io/services/development-process.html) in the documentation.
+
+## Developing
+
+This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services](https://docs.kalix.io/services/) and check Spring-SDK [official documention](https://docs.kalix.io/spring/index.html). Examples can be found [here](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples) in the folders with "spring" in their name.
```
Comment (efgpinto) [inline maven-java/kalix-spring-boot-kotlin-archetype/src/main/resources/archetype-resources/README.md:9]: My bad 😬 

```suggestion
This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services](https://docs.kalix.io/services/) and check Spring-SDK [official documentation](https://docs.kalix.io/spring/index.html). Examples can be found [here](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples) in the folders with "spring" in their name.
```

**Review by efgpinto — APPROVED** [review]
LGTM again, and sorry for the typo 🤦‍♂️

**Pull request merged**